AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/ActivityAdminController.cs

```diff
@@ -71,13 +85,8 @@ public IActionResult Details(int id)
                 Description = activity.Description,
                 StartDateTime = activity.StartDateTimeUtc,
                 EndDateTime = activity.EndDateTimeUtc,
-                Volunteers = _dataAccess.ActivitySignups.Where(asup => asup.Activity.Id == id).Select(u => u.User.UserName).ToList(),
```

Comment:
There was an extra query happening here that wasn't needed. This data is already included